docs/how-does-it-work.md

```diff
@@ -1,21 +1,12 @@
 ## How does Okteto work?
 
-This is how a standard dev environment looks like:
+Okteto swaps your applications pods by a development environment:
 
-<img align="left" src="env.png">
+<img align="left" src="okteto-architecture.png">
 
-&nbsp;
+The **api-dev** pod duplicates the manifest of the **api** pod, so it is fully integrated with your Kubernetes setup.
```

Comment:
this is not obvious a lot. I think you should mention how it's fully integrated.